common/generate_elastic_template.py

```diff
@@ -0,0 +1,96 @@
+"""
+Generate an elastic scattering input template for Fresco.
+"""
+
+from fractions import Fraction
+from pathlib import Path
+
+from ._utils import _is_fraction_integer_or_half_integer
+
+
+def load_template(file_path: Path) -> str:
+    with open(file_path, "r") as file:
+        return file.read()
+
+
+template_file_path = Path(__file__).parent / "templates/elastic.template"
+with open(template_file_path, "r") as file:
+    elastic_input_template = file.read()
```

Comment:
`template_file_path` is a constant for this file.  One could argue that the contents of `elastic_input_template` are also effectively a constant.

Put variable names in all caps to communicate that?  Leading underscore to emphasize that they are internal to file?